Userland/Libraries/LibAudio/Buffer.h

```diff
@@ -77,6 +77,10 @@ class Buffer : public RefCounted<Buffer> {
     {
         return adopt_ref(*new Buffer(move(buffer), buffer_id, sample_count));
     }
+    static NonnullRefPtr<Buffer> create_empty()
+    {
+        return adopt_ref(*new Buffer({}));
```

Comment:
With the recent move towards `TRY()` and handling alloc failures in userspace, this is a little bit of a bad idea
since it's a new API, it'd be nice to at least make it `ErrorOr<NNRP<Buffer>>`...